eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/mob/ButcherArgument.java

```diff
@@ -16,15 +16,13 @@
 import dev.rollczi.litecommands.suggestion.SuggestionContext;
 import dev.rollczi.litecommands.suggestion.SuggestionResult;
 import org.bukkit.command.CommandSender;
-import org.bukkit.entity.Player;
 
-import java.util.UUID;
 import java.util.stream.IntStream;
 
 @LiteArgument(type = int.class, name = ButcherArgument.KEY)
 class ButcherArgument extends AbstractViewerArgument<Integer> {
 
-    static final String KEY = "butcher";
```

Comment:
ino zmiana na minus, tego nie trzeba zmieniać na chunks bo to wsm nic nie reprezentuje

Reply:
First argument of the `/butcher` command is number of chunks that are cleared (or at least it should work like that), in my opinion looks a lot better with <chunks>

Reply:
RECE OPADAJA, klucz to klucz a nie wyświetlana nazwa argumentu

Reply:
> RECE OPADAJA, klucz to klucz a nie wyświetlana nazwa argumentu

In the 3.0 LiteCommands using @Arg(ArgumentKey) lite-commands shows the key as display name in the invalid usage, if you want don't use this feature, you can use the @Key Annotation

Reply:
I tip my hat angole

Reply:
> I tip my hat angole

Lite Developerka tak zwana